eq/deflect/eventHandler.cpp

```diff
@@ -178,80 +169,97 @@ void EventHandler::_processEvents( const Proxy* proxy )
         {
         case ::deflect::Event::EVT_KEY_PRESS:
         case ::deflect::Event::EVT_KEY_RELEASE:
-            event.type = deflectEvent.type == ::deflect::Event::EVT_KEY_PRESS ?
-                                          Event::KEY_PRESS : Event::KEY_RELEASE;
-            event.keyPress.key = _getKey(  deflectEvent.key );
-            break;
+        {
+            KeyEvent event;
+            event.originator = channel.getID();
+            event.serial = channel.getSerial();
+            const EventType type =
+                deflectEvent.type == ::deflect::Event::EVT_KEY_PRESS ?
+                                         EVENT_KEY_PRESS : EVENT_KEY_RELEASE;
+            event.key = _getKey(  deflectEvent.key );
+            channel.processEvent( type, event );
+            return;
```

Comment:
even though that's what the old code was doing with 'break', I'm wondering it should be 'continue' to handle all pending events?